platform/ios/MGLMapView.mm

```diff
@@ -383,6 +383,40 @@ - (void)commonInit
     }];
 }
 
+- (NS_ARRAY_OF(NSDictionary *) *)featuresAt:(CGPoint)point
+{
+    // flip y for core
+    point.y = self.bounds.size.height - point.y;
```

Comment:
To avoid both this flip and having to pass the `TransformState` into the style, we should call `-convertPoint:toCoordinateFromView:` here and pass a coordinate in.